Pds4File/pds4_rules/__init__.py

```diff
@@ -0,0 +1,10 @@
+################################################################################
+# rules/__init__.py
```

Comment:
`pds4_rules/__init__.py`
Because we've decided to work on rules for different instruments (covims or coiss), let's either remove the file or put a comment here saying that this rule is currently a draft  and we will work on it later.

Reply:
Currently `pds4_rules/__init__.py` contains:
```
  __all__ = [
       "uranus_occs_earthbased",
       "cassini_iss",
      "cassini_vims",
  ]
```

Reply:
I misunderstood, sorry I thought you were referring to the entire file (__init__.py) but realise you meant the uranus occs rules (I've added a comment saying we will work on this later)